department-of-veterans-affairs/abd-vro
# fix spring issue

## What was the problem?
facing `java.lang.NoSuchFieldError: TYPE_HIERARCHY_AND_ENCLOSING_CLASSES` issue in `svc-bie-kafka` service due to using latest kafka version


## How does this fix it?[^1]
revert the version bump change back and will use `2.8.11` for now.  

## How to test this PR
- Run `3. Continuous Integration (run tests and checks)` github workflow and ensure all steps are passing


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
